tests/migrations/test_commands.py

```diff
@@ -463,6 +463,15 @@ def test_sqlmigrate_forwards(self):
             "Transaction end not found or found before operation description (unique_together)"
         )
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_db_constraint"})
+    def test_dbconstraint_alter_field_migration(self):
+        out = io.StringIO()
+        call_command("sqlmigrate", "migrations", "0002", stdout=out)
+        output = out.getvalue().lower()
+
+        index_add_constraint = output.find('add constraint')
+        self.assertEqual(index_add_constraint, -1)
```

Comment:
I only wrote it this way for consistency with the other tests that parse the string output.